StardewArchipelago/GameModifications/EntranceRandomizer/EntranceManager.cs

```diff
@@ -176,6 +176,7 @@ public bool TryGetEntranceReplacement(string currentLocationName, string locatio
             // return false;
             if (!TryGetModifiedWarpName(key, out var desiredWarpName))
             {
+                _monitor.Log($"Tried to find warp from {currentLocationName} but found none.  Giving default warp.", LogLevel.Error);
```

Comment:
This should definitely not be on LogLevel.Error. It is, at best info, but probably trace, as it is expected to go through this code many, many times, without it being a problem, over the course of gameplay